### ISSUE TITLE
Detect / Point To Already Installed Software

### ISSUE THREAD

**Issue by zachbugay** [microsoft/devhome#898]
### Suggested new feature or improvement

I have already installed software via Scoop / WinGet. I would like to point to already installed software. 

### Scenario

I have already setup my machine, and installed all necessary applications. I installed Dev Home after the fact. 

### Additional details

I've setup my machine already. Once Dev Home is installed, point to previously installed software.

**Comment by cinnamon-msft** [comment]
When you say point to installed software, do you mean in the machine configuration -> install applications tool? Or would you like to see them somewhere else?

**Comment by denelon** [comment]
Related to:
* #20

I've been thinking about having essentially "full" support of WinGet and the various features exposed by package managers. The ability to install specific versions of tools, pinning, etc.

**Comment by zachbugay** [comment]
> When you say point to installed software, do you mean in the machine configuration -> install applications tool? Or would you like to see them somewhere else?

Yes, when I see the "Machine configuration" section, it is telling me to install Git 2.41.0 through Winget. However, I've already installed Git into my user space, via scoop.

**Comment by michael-hawker** [comment]
Looks like it can't differentiate between VS editions either:

![image](https://github.com/microsoft/devhome/assets/24302614/27d33838-7fc3-4c5d-9e8f-e667c623622b)
